src/machine/usb_nxpmk66f18.go

```diff
@@ -0,0 +1,626 @@
+// +build nxp,mk66f18
+
+package machine
+
+import (
+	"device/nxp"
+	"fmt"
```

Comment:
Don't import fmt in the machine package. This is a _huge_ dependency with a large dependency graph and importing it in the machine package may lead to circular dependencies. It also bloats the binary and increases compile time.

Instead, I would suggest to use `println` for debugging (and disabled by default).